src/modules/ModuleFarming/composable.pools.ts

```diff
@@ -128,11 +130,11 @@ export function useFilteredPools<T extends Pool>(
 function comparePools<T extends Pool>(poolA: T, poolB: T, sorting: Sorting): number {
   switch (sorting) {
     case Sorting.Hot:
-      return poolB.annualPercentageRate.comparedTo(poolA.annualPercentageRate)
+      return (poolB.annualPercentageRate ?? new BigNumber(0)).comparedTo(poolA.annualPercentageRate ?? new BigNumber(0))
```

Comment:
please create this const above (before switch or somewhere in consts) - `new BigNumber(0)`

Reply:
Done